format/index.md

```diff
@@ -903,11 +902,20 @@ Accept: application/vnd.api+json
   "data": {
     "type": "photos",
     "title": "Ember Hamster",
-    "src": "http://example.com/images/productivity.png"
+    "src": "http://example.com/images/productivity.png",
+    "links": {
+      "photographer": {
+        "linkage": { "type": "people", "id": "9" }
+      }
+    }
   }
 }
 ```
 
+If the resource object being sent contains any relationships in its `"links"`
+key, those relationships **MUST** each have a `"linkage"` key that includes the
+linkage the new resource is to have.
+
```

Comment:
This phrasing seems awkward. How about keeping it similar to the note below for PATCH:

> If a relationship is provided in the `links` section of a resource object, its value **MUST** be a link object with a `linkage` member.